deploy/yamls/ubuntu/zfs-driver.yaml

```diff
@@ -701,6 +701,24 @@ roleRef:
 
 ---
 
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: openebs-zfspv-bin
+  namespace: kube-system
```

Comment:
@pawanpraka1 can you add comments as to why this should be in kube-system namespace.

Reply:
added @kmova